packages/eslint-plugin-lit-a11y/lib/utils/aria.js

```diff
@@ -0,0 +1,24 @@
+const { aria, roles } = require('aria-query');
+
+/**
+ * @param {string} string
+ * @return {string is import("aria-query").ARIARoleDefintionKey}
+ */
+function isAriaRole(string) {
+  // @ts-expect-error: we need to disambiguate from the string type here.
+  return [...roles.keys()].includes(string);
```

Comment:
should this maybe `.get`, like below, for perf?

Reply:
sounds good 👍